frontend/model/contracts/group.js

```diff
@@ -1409,7 +1418,7 @@ sbp('chelonia/defineContract', {
       sideEffect ({ meta, data, contractID, innerSigningContractID }, { state }) {
         const rootState = sbp('state/vuex/state')
         const memberID = data.memberID || innerSigningContractID
-        if (innerSigningContractID === rootState.loggedIn.identityContractID && !sbp('okTurtles.data/get', 'JOINING_GROUP-' + contractID)) {
```

Comment:
Can you explain here on GH (and maybe in a comment) why this `'JOINING_GROUP-' + contractID` check is no longer necessary?